recipe/meta.yaml

```diff
@@ -1,57 +1,57 @@
 {% set version = "0.18.2" %}
-
 {% set variant = "openblas" %}
 
 package:
   name: scikit-learn
   version: {{ version }}
 
 source:
-  fn: scikit-learn-{{ version }}.tar.gz
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
   sha256: fc4de6feaa6682801abac19f8e4f3ed0ea56b39e7e5a89ab66c7e0a76f661445
   patches:
     # TODO: Remove this patch when 0.19 is released
     - skip-tests-failing-on-0.18.2.patch
 
 build:
-  number: 200
+  number: 201
   # We lack openblas on Windows, and therefore can't build numpy there either currently.
-  skip: true  # [win or np!=113]
+  skip: true  # [win]
   features:
     - blas_{{ variant }}
 
 requirements:
   build:
-    - cython
     - python
+    - cython
     - setuptools
     - blas 1.1 {{ variant }}
     - openblas 0.2.19|0.2.19.*
-    - numpy x.x
+    - numpy 1.8.*  # [not (win and py36)]
+    - numpy 1.11.*  # [win and py36]
```

Comment:
> Actually the Windows line shouldn't even be here if we are still skipping building on Windows.

You are the first one to ensure first as future proof. I don't see why not leave them there. Numpy with openblas for Windows will be coming to conda-forge sooner than you think.